src/index.tsx

```diff
@@ -1,99 +1,19 @@
-import React, { FC, ReactNode, useEffect, useRef } from "react";
+import { ReactNode, useEffect, useRef } from "react";
 import * as ReactDOM from "react-dom";
-import { ChatView } from "components/ChatView";
+import { Props, ChatView } from "./components/ChatView";
+import type { SummaryLanguage } from "./types";
+export type { Props } from "components/ChatView";
+export { DEFAULT_SUMMARIZER } from "./useChat";
 
 // @ts-ignore
 import cssText from "index.scss";
-import type { SummaryLanguage } from "./types";
-
-export interface Props {
-  // Vectara customer ID
-  customerId: string;
-
-  // Vectara API key
-  apiKey: string;
-
-  // Vectara corpus IDs
-  corpusIds: string[];
-
-  // Title to be shown in the UI header
-  title?: string;
-
-  // Text to be shown when the input field has no text
-  placeholder?: string;
-
-  // Example questions to prompt the user
-  exampleQuestions?: string[];
-
-  // Size of input. Defaults to "large".
-  inputSize?: "large" | "medium";
-
-  // Content to render into the messages display when there are no messages to show
-  emptyStateDisplay?: ReactNode;
-
-  // Used to explicitly configure the component's initial open/closed state.
-  isInitiallyOpen?: boolean;
-
-  // Used to control the component's z-index. Defaults to 9999.
-  zIndex?: number;
-
-  // Used to enable streaming responses from the API. Defaults to true.
-  enableStreaming?: boolean;
-
-  // The language the responses should be in. Defaults to English.
-  language?: SummaryLanguage;
-}
-
-/**
- * A client-side chat component that queries specific corpora with a user-provided message.
- */
-const ReactChatbotInternal: FC<Props> = ({
```

Comment:
This didn't seem needed because it seemed like all it added was a wrapper `div`. Did I overlook something or is this change OK?

Reply:
I see. I think I'm thinking of `src/components/ChatView.tsx` as the internal chatbot. Let's keep this change for now but I'm open to talking it over if you feel strongly.